src/index.html

```diff
@@ -5,8 +5,359 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="container">
+      <a href="#home" class="logo">
+        <img src="images/logo.png" alt="no">
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a href="#" class="nav_linka">
+              Apple
+            </a>
+          </li>
+          <li class="nav_item"><a href="#" class="nav_link">Samsung</a></li>
+          <li class="nav_item"><a href="#" class="nav_link">Smartphone</a></li>
+          <li class="nav_item">
+            <a href="#" class="nav_link" 
+            data-qa="nav-hover">Laptops & Computers</a>
+          </li>
+          <li class="nav_item"><a href="#" class="nav_link">Gadgets</a></li>
+          <li class="nav_item"><a href="#" class="nav_link">Tablets</a></li>
+          <li class="nav_item"><a href="#" class="nav_link">Photo</a></li>
+          <li class="nav_item"><a href="#" class="nav_link">Video</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
```

Comment:
u don't need to create block for each line, just create common container for them, it will be enough